docs/basics/notice.md

```diff
@@ -0,0 +1,14 @@
+<!--
+order: 7
+-->
+
+# Notice
+
+Important information for Etheremint development.
+
+## Cross-Chain Transaction Replay Attack Prevention
+
+Given the fact that Ethermint uses the same transaction signature pattern and the type of chainIDEpoch for Ethereum compatibility, 
```

Comment:
maybe reword to say "Given the fact that Ethermint uses the same transaction signature format as Ethereum for compatibility with Ethereum tooling, and Ethermint's  `chainIDEpoch` corresponds to the chain ID on Ethereum, any valid transaction..."